src/main/java/com/getbase/sync/SyncProcess.java

```diff
@@ -19,27 +20,41 @@
     private Session session;
     private List<Map<String, Object>> nextItems;
 
+    private long timeToRunInMillis;
+
     public SyncProcess(Client client, String deviceUUID, SessionManager sessionManager,
-                       BiPredicate<Meta, Map<String, Object>> predicate) {
+                       BiPredicate<Meta, Map<String, Object>> predicate,
+                       long timeToRunInMillis) {
```

Comment:
Are you sure that this change won't break backward compatibility? Is this possible that some customer use this class directly, not via `Sync`?

Reply:
I am not 100% sure that no-one uses it so we will add extra ctor to match previous signature.

@tomasz-rojek could we add the extra ctor for backward compatibility?